pkg/users/account-create-dialog.js

```diff
@@ -92,16 +92,13 @@ function AccountCreateBody({ state, errors, change, shells }) {
 
             <FormGroup label={_("Shell")}
                        fieldId="accounts-create-user-shell">
-                <Select variant={SelectVariant.single}
-                        toggleId="accounts-create-user-shell"
-                        onToggle={statusIsExpanded => change("isShellSelectExpanded", statusIsExpanded)}
-                        onSelect={(event, selection) => { change("shell", selection); change("isShellSelectExpanded", false) }}
-                        selections={shell}
-                        isOpen={isShellSelectExpanded}
-                        aria-labelledby="vm-state-select"
-                        menuAppendTo="parent">
-                    { shells.map(shell_path => <SelectOption value={shell_path} key={shell_path} label={shell_path}>{shell_path}</SelectOption>) }
-                </Select>
+                <FormSelect
+                        data-selected={shell}
+                        id="accounts-create-user-shell"
+                        onChange={selection => { change("shell", selection) }}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18671-20230419-094958-5a71cc2f-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)